pages/doc.js

```diff
@@ -32,57 +31,66 @@ export default class Documentation extends Component {
       markdown: '',
       headings: [],
       pageNotFound: false,
-      isMenuOpen: false
+      isMenuOpen: false,
+      load: false
     }
   }
 
   componentDidMount() {
     this.loadStateFromURL()
     this.initDocsearch()
     window.addEventListener('popstate', this.loadStateFromURL)
-    this.ps = new PerfectScrollbar('#sidebar-menu', {
```

Comment:
why is it removed?

Reply:
we did not see the difference in the work without this plugin, and therefore turned it off

Reply:
no, they have different behavior. The current implementation scrolls smoother everything when you scroll on left-side menu. Try to scroll aggressively - you should see the difference. Also, I would check if IE was a major reason for this. In general, please mention that you are removing something. In most cases, there is a reason for stuff to exist.

Reply:
we tried to aggressively scroll - but did not see the difference.

Reply:
fixed